gobiko/apns/client.py

```diff
@@ -213,16 +208,11 @@ def _send_push_request(self, connection, registration_id, json_data, request_hea
 
         if response.status != APNSResponse.Success:
             body = json.loads(response.read().decode('utf-8'))
-            reason = body["reason"] if "reason" in body else None
+            reason = body.get("reason")
 
             if reason:
                 exceptions_module = importlib.import_module("gobiko.apns.exceptions")
-                ExceptionClass = None
```

Comment:
This is redundant assignment, because `try except` blocks in python don't introduce a new scope. Actually, my IDE highlights this line as unused, because `ExceptionClass` is redefined in both branches of `try except`